apps/web/src/components/Battle/BattleHeader.tsx

```
@@ -93,116 +30,27 @@ function BattleHeader({ theme, toggleTheme, onLeaveClick }: BattleHeaderProps) {
 
       <div className="flex items-center gap-3">
-        <div className="inline-flex items-center gap-1.5 rounded-full bg-base-faint px-5 py-2 text-sm font-medium text-base-primary">
-          <Eye className="h-4 w-4 text-base-secondary" strokeWidth={2.5} />
-          <span className="text-base-primary">{spectatorCount}</span>
-        </div>
+        <SpectatorCountBadge />
         <button
           onClick={onLeaveClick}
           className="inline-flex h-10 w-20 items-center justify-center rounded-full bg-base-faint text-sm font-bold text-base-primary transition hover:brightness-110"
         >
           나가기
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)